src/current/v23.1/molt-verify.md

```diff
@@ -31,64 +31,74 @@ The following databases are currently supported:
 
 ## Use the MOLT Verify tool
 
-1. Get the JDBC connection strings for the source and target databases you want to compare.
-1. Make sure the SQL user running MOLT Verify has read privileges on the necessary tables.
-1. From the [Releases](https://github.com/cockroachdb/molt/releases/) page, download the binary that matches your system:
+1. [Download the binary](https://github.com/cockroachdb/molt/releases/) that matches your system:
   - For Mac: `molt.darwin.amd64` for Intel or `molt.darwin.arm64` for ARM
   - For Windows: `molt.amd64.exe`
   - For Linux: `molt.linux.amd64`
    
     Rename the binary to `molt` and add it to your `PATH` so you can execute the `molt verify` command from any shell.
+
+1. Get the connection strings for the source and target databases you want to compare.
```

Comment:
I think we only will support CRDB as a target for the forseeable future. Maybe we can just link to https://www.cockroachlabs.com/docs/stable/connect-to-the-database ?

Reply:
Good point; done.